### DISCUSSION TITLE
请问控制台该如何打开？

### DISCUSSION THREAD

**Discussion by musearcosu** [ikaros-dev/ikaros#528] [Q&A] [answered]
用谷歌浏览器输入/console的话是搜索，用edge显示无法打开

**Comment by musearcosu** [comment]
补：使用 Docker Compose 部署

**Reply by chivehao** [reply]
ip:port/console

**Comment by chivehao** [comment]
> 用谷歌浏览器输入/console的话是搜索，用edge显示无法打开

请问是否有清理过浏览器缓存？

能否提供问题截图，和浏览器F12控制台日志截图？

**Comment by chivehao** [comment]
`http://ip(domain):port/console`

**Reply by musearcosu** [reply]
请问ip和端口号是在哪获取？我还以为指的就是直接输/console。。。

**Reply by chivehao** [reply, marked as answer]
从你的部署环境获取，你是部署在哪台机器上，IP就是那个机器的IP地址，可能是局域网IP，也可能是公网IP，端口没修改的话默认就是9999

**Reply by musearcosu** [reply]
原来如此，可以打开了，谢谢！
